Possible go module version conflict (github.com/shirou/gopsutil)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

A bug happened!

Attempting to update omp via Homebrew, running into this install error:

```shell
% HOMEBREW_GOPROXY=direct brew upgrade
==> Upgrading 1 outdated package:
jandedobbeleer/oh-my-posh/oh-my-posh 12.25.1 -> 12.28.1
==> Fetching jandedobbeleer/oh-my-posh/oh-my-posh
==> Downloading https://github.com/JanDeDobbeleer/oh-my-posh/archive/v12.28.1.tar.gz
Already downloaded: /Users/gavinaws/Library/Caches/Homebrew/downloads/d7d1d436a8b714a0e3bb8d0895adca105a8c343bddf8c86a9d32ea1e63a38da0--oh-my-posh-12.28.1.tar.gz
==> Upgrading jandedobbeleer/oh-my-posh/oh-my-posh
  12.25.1 -> 12.28.1

==> go build -o=oh-my-posh -ldflags="-s -w -X 'main.Version=12.28.1'"
Last 15 lines from /Users/gavinaws/Library/Logs/Homebrew/oh-my-posh/01.go:
2022-12-21 17:55:56 +0000

go build -o=oh-my-posh -ldflags="-s -w -X 'main.Version=12.28.1'"

go: downloading github.com/shirou/gopsutil/v3 v3.22.11
segments/sysinfo.go:7:2: reading github.com/shirou/gopsutil/go.mod at revision v3.22.11: unknown revision v3.22.11
segments/sysinfo.go:8:2: reading github.com/shirou/gopsutil/go.mod at revision v3.22.11: unknown revision v3.22.11
platform/shell_unix.go:11:2: reading github.com/shirou/gopsutil/go.mod at revision v3.22.11: unknown revision v3.22.11
segments/sysinfo.go:9:2: reading github.com/shirou/gopsutil/go.mod at revision v3.22.11: unknown revision v3.22.11
segments/sysinfo.go:10:2: reading github.com/shirou/gopsutil/go.mod at revision v3.22.11: unknown revision v3.22.11
platform/shell.go:26:2: reading github.com/shirou/gopsutil/go.mod at revision v3.22.11: unknown revision v3.22.11

Do not report this issue to Homebrew/brew or Homebrew/core!
```

For reference, my company blocks access to goproxy, so the `HOMEBREW_GOPROXY=direct brew ...` is needed to go directly to dependencies.

The reason I think it may be version related is that the most recent version at https://github.com/shirou/gopsutil is `v3.22.10`.

Checking to see if Oh-my-posh related change or something else!

### Theme

No theme related issues.

### What OS are you seeing the problem on?

macOS

### Which shell are you using?

zsh

### Log output

```Shell
I'd love to, but this is related to the upgrade. After the failed upgrade, `oh-my-posh` is no longer installed on my system.
```

--- a/src/go.mod
+++ b/src/go.mod
@@ -19,7 +19,7 @@ require (
 	github.com/mitchellh/copystructure v1.2.0 // indirect
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/sergi/go-diff v1.2.0 // indirect
-	github.com/shirou/gopsutil/v3 v3.22.11
+	github.com/shirou/gopsutil/v3 v3.22.10
 	github.com/stretchr/objx v0.5.0 // indirect
 	github.com/stretchr/testify v1.8.1
 	github.com/wayneashleyberry/terminal-dimensions v1.1.0
@@ -92,4 +92,4 @@ require (
 
 replace github.com/atotto/clipboard v0.1.4 => github.com/jandedobbeleer/clipboard v0.1.4-1
 
-replace github.com/shirou/gopsutil/v3 v3.22.9 => github.com/jandedobbeleer/gopsutil/v3 v3.22.9-2
+replace github.com/shirou/gopsutil/v3 v3.22.10 => github.com/jandedobbeleer/gopsutil/v3 v3.22.10-1

--- a/website/docs/segments/az.mdx
+++ b/website/docs/segments/az.mdx
@@ -59,3 +59,4 @@ initializing Oh My Posh.
 | `.Origin`          | `string`  | where we received the information from, can be `CLI` or `PWSH` |
 
 [templates]: /docs/configuration/templates
+[az]: https://www.powershellgallery.com/packages/Az

